controls/cis_ocp_1_3_0/section-3.yml

```diff
@@ -6,27 +6,42 @@ controls:
   controls:
   - id: '3.1'
     title: Authentication and Authorization
-    status: pending
+    status: automated
     rules: []
     controls:
+    # FIXME(rhmdnd): This control has some typos in the audit steps that cause
+    # them to not work properly (specifically, escaping characters). This needs
+    # to be fixed in CIS upstream.
```

Comment:
Fixing this in https://workbench.cisecurity.org/sections/1652513/recommendations/3216956